.github/dependabot.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+updates:
+
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 5
+    labels: ["component/dependencies"]
+
+  # Maintain dependencies for dashboard, shell, and packages
+  - package-ecosystem: "npm"
+    directories:
+      - "/"
```

Comment:
minor comment, we list the root but also directories within the root. is this because dependabot looks for a package.json within just the root of the directories listed here? if so we might want to also cover the package.json files in the creators, possibly also docusaurus and storybook?

Reply:
That's correct, you can see this blog post from April that describes the behavior quite succinctly: https://github.blog/changelog/2024-04-29-dependabot-multi-directory-configuration-public-beta-now-available/